test-context/src/lib.rs

```diff
@@ -27,6 +27,11 @@ pub struct TrustifyContext {
     postgresql: Option<PostgreSQL>,
 }
 
+use peak_alloc::PeakAlloc;
+
+#[global_allocator]
+static PEAK_ALLOC: PeakAlloc = PeakAlloc;
```

Comment:
This looks useful full-time/always, yeah?

Reply:
Maybe. I hadn't intended on leaving it in, but I'll reconsider.

Reply:
Further, it'll be mostly useless unless you `--test-threads=1`